### DISCUSSION TITLE
Item descriptions of deusizine and saline are reversed in Simplified Chinese & Traditional Chinese

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#14219] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

SC
<entitydescription.deusizine>输液用氯化钠溶液，对治疗失血症状有一定效用。</entitydescription.deusizine>
<entitydescription.antibloodloss1>一种强效兴奋剂，能有效治疗内脏损伤、缺氧与少量失血，但存在造成轻微灼伤的副作用。</entitydescription.antibloodloss1>

TC
<entitydescription.deusizine>輸液用氯化鈉溶液，對治療失血症狀有一定效用。</entitydescription.deusizine>
<entitydescription.antibloodloss1>一種強效興奮劑，能有效治療內臟損傷、缺氧與少量失血，但存在造成輕微灼傷的副作用。</entitydescription.antibloodloss1>

EN
<entitydescription.deusizine>A highly potent stimulant effective in the treatment of internal damage, oxygen deprivation, blood loss and, to a lesser extent, bleeding. Produces mild burns as a side effect.</entitydescription.deusizine>
<entitydescription.antibloodloss1>A sodium chloride infusion mildly useful in the treatment of blood loss. Small relief when used on infected tissue.</entitydescription.antibloodloss1>

### Reproduction steps

Check text files in SC&TC

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've now fixed the issue in our private repo, the fix should be included in the next release.
